src/tests/test_cases/test.py

```diff
@@ -368,13 +374,13 @@ def docstring_bad_ignore_all():  # noqa
     pass
 
 
-def docstring_bad_ignore_one():  # noqa: D400,D401
+def docstring_bad_ignore_one():  # noqa: D400,D401,D415
     """Runs something"""
     pass
 
 
 @expect("D401: First line should be in imperative mood ('Run', not 'Runs')")
-def docstring_ignore_violations_of_pydocstyle_D400_and_PEP8_E501_but_catch_D401():  # noqa: E501,D400
+def docstring_ignore_violations_of_pydocstyle_D400_D415_and_PEP8_E501_but_catch_D401():  # noqa: E501,D400,D415
```

Comment:
This is getting out of hand :)
Maybe change to `ignore_some_violations_but_catch_D401`?